posts/2022-04-07-Rust-1.60.0.md

```diff
@@ -0,0 +1,172 @@
+
+If you have a previous version of Rust installed via rustup, you can get 1.60.0 with:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.60.0][notes] on GitHub.
+If you'd like to help us out by testing future releases, you might consider updating locally to use
+the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report] any bugs you might come across!
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1600-2022-04-07
+[report]: https://github.com/rust-lang/rust/issues/new/choose
+
+## What's in 1.60.0 stable
+
+### Instrumentation-based Code Coverage
+
+Support for LLVM-based coverage instrumentation has been stabilized in rustc, enabled with `-C instrument-coverage`. You can try this out on your code by rebuilding your code with `-Cinstrument-coverage`:
+
+```shell=
+RUSTFLAGS="-C instrument-coverage" cargo build
+```
+
+After that, you can run the resulting binary, which will produce
+a `default.profraw` file in the current directory. That can be consumed
+by the `llvm-cov` binary shipped with rustc as part of the llvm-tools-preview
+component.
+
+```shell=
+rustup component add llvm-tools-preview
+$(rustc --print sysroot)/lib/rustlib/x86_64-unknown-linux-gnu/bin/llvm-profdata merge -sparse default.profraw -o default.profdata
+$(rustc --print sysroot)/lib/rustlib/x86_64-unknown-linux-gnu/bin/llvm-cov show -Xdemangler=rustfilt target/debug/coverage-testing \
+    -instr-profile=default.profdata \
+    -show-line-counts-or-regions \
+    -show-instantiations
+```
+
+The above commands on a simple helloworld binary produce this annotated report, showing that each line of the input was covered.
+
+```
+    1|      1|fn main() {
+    2|      1|    println!("Hello, world!");
+    3|      1|}
+```
+
+For more details, please read the [documentation](https://doc.rust-lang.org/rustc/instrument-coverage.html) in the rustc book. The specific output format and LLVM tooling to consume it are both not guaranteed to exist in this specific form, but the baseline functionality will continue to exist for future Rust releases.
+
+### `cargo --timings`
+
+Cargo has stabilized support for collecting information on build with the `--timings` flag.
+
+```shell
+$ cargo build --timings
+   Compiling hello-world v0.1.0 (hello-world)
+      Timing report saved to target/cargo-timings/cargo-timing-20220318T174818Z.html
+    Finished dev [unoptimized + debuginfo] target(s) in 0.98s
+```
+
+The report is also copied to `target/cargo-timings/cargo-timing.html`. A report on the release build of Cargo has been put up [here](/images/2022-04-07-timing.html). These reports can be useful for improving build performance.
+
```

Comment:
Just a suggestion, perhaps it might be helpful for people to find more information.  

```suggestion
More information about the timing reports may be found in the [documentation](https://doc.rust-lang.org/nightly/cargo/reference/timings.html).

```